test/unit_tests/stack.jl

```diff
@@ -0,0 +1,38 @@
+using AbstractPlotting: stack_grouped_from_to
+
+@testset "grouped bar: stack" begin
+	x1         = [1, 1,  1,  1]
+    grp_dodge1 = [2, 2,  1,  1]
+	grp_stack1 = [1, 2,  1,  2]
+    y1         = [2, 3, -3, -2]
+    
+	x2         = [2, 2,  2,  2]
+	grp_dodge2 = [3, 4,  3,  4]
+	grp_stack2 = [3, 4,  3,  4]
+    y2         = [2, 3, -3, -2]
```

Comment:
Formatting got a bit off here and below.

Reply:
fixed